src/com/sap/piper/DownloadCacheUtils.groovy

```diff
@@ -0,0 +1,49 @@
+package com.sap.piper
+
+
+class DownloadCacheUtils {
+
+    static boolean isEnabled(Script script) {
+        script.node('master') {
+            String network = script.env.DL_CACHE_NETWORK
+            String host = script.env.DL_CACHE_HOSTNAME
+            return (network.asBoolean() && host.asBoolean())
+        }
+    }
+
+    static String getDockerOptions(Script script) {
+        script.node('master') {
```

Comment:
Why in master?

Reply:
It should also be available in any agent. At least we should check that.

Reply:
only on the master node we have those env vars

Reply:
But then we have to make sure that this it not executed in the kubernetes context, which can also be done somewere else. There might not be a master agent on Kubernetes.

Reply:
Yes. 
I would use this util class in a way like this:
```groovy
if(!env.ON_K8S)
  DownloadCacheUtils....
```